python/ray/tests/test_cancel.py

```diff
@@ -129,7 +129,7 @@ def wait_for(t):
 def test_comprehensive(ray_start_regular, use_force):
     signaler = SignalActor.remote()
 
-    @ray.remote
+    @ray.remote(num_cpus=0)
```

Comment:
Can we remove this now?

Reply:
Same as the above test, but I don't think so, because there are multiple `wait_for` tasks and they can't run while the other is blocked on `ray.get()` now